app/components/card_list/index.js

```diff
@@ -125,7 +125,11 @@ class CardList extends Component {
                             style={[styles.button, styles.cardButton, styles.cardButtonWallet]}
                             backgroundColor={COLORS[index % COLORS.length]}
                         >
-                            <TouchableOpacity onPress={() => Actions.card_view({card: item})} onLongPress={() => this.showOptions(item)}>
+                            <TouchableOpacity
```

Comment:
Another possible solution would be to move the view above inside of TouchableOpacity. Since TouchableOpacity wraps its children, it would be the same size as the colored label. In this way you wouldn't need to specify a style for TouchableOpacity.

Reply:
That's what I was trying to do at first, but for some reason the background color wouldn't show up when I did that.